src/browser-frontend/reducers/pages-reducers.js

```diff
@@ -24,12 +24,14 @@ function addPage(state, { payload: { url, parentId, background } = {} }) {
     const pageId = uuid();
     const pageUrl = url || Endpoints.DEFAULT_PAGE_URL;
 
+    const tabOwner = parentId || '';
```

Comment:
No need to `|| ''` here, the default is set by the Immutable.Record (which in this case would be null).